docs/getting-started-guides/coreos/azure/README.md

```diff
@@ -0,0 +1,187 @@
+# Kubernetes on Azure with CoreOS and [Weave](http://weave.works)
+
+## Introduction
+
+In this guide I will demonstrate how to deploy a Kubernetes cluster to Azure cloud. You will be using CoreOS with Weave, which implements simple and secure networking, in a transparent, yet robust way. The purpose of this guide is to provide an out-of-the-box implementation that can ultimately be taken into production with little change. It will demonstrate how to provision a dedicated Kubernetes master and etcd nodes, and show how to scale the cluster with ease.
+
+## Let's go!
+
+To get started, you need to checkout the code:
+```
+git clone https://github.com/GoogleCloudPlatform/kubernetes
```

Comment:
can you change this to explicitly pull the v0.11.0 tag?  That way, if upstream changes, your guide will keep working until you next update and test i.t